conda/core/portability.py

```diff
@@ -13,7 +13,7 @@
 from ..auxlib.ish import dals
 from ..base.constants import PREFIX_PLACEHOLDER
 from ..base.context import context
-from ..common.compat import on_win, on_linux
+from ..common.compat import on_win, on_linux, on_mac
```

Comment:
Should be able to remove both of these now:

```suggestion
from ..common.compat import on_win
```